src/log/log.c

```diff
@@ -1074,6 +1074,46 @@ err:	__wt_scr_free(session, &buf);
 	return (ret);
 }
 
+/*
+ * __log_record_verify --
+ *	Check that values of the log record header are valid.
+ */
+static int
+__log_record_verify(WT_SESSION_IMPL *session, WT_FH *log_fh, uint32_t offset,
+    WT_LOG_RECORD *logrec, bool *corrupt)
+{
+	size_t i;
+
+	*corrupt = false;
+	if (F_ISSET(logrec, ~(WT_LOG_RECORD_ALL_FLAGS))) {
+		WT_RET(__wt_msg(session,
+		    "%s: log record at position %" PRIu32
+		    " has flag corruption", log_fh->name, offset));
+		*corrupt = true;
+	}
+	for (i = 0; i < sizeof(logrec->unused); i++)
+		if (logrec->unused[i] != 0) {
+			WT_RET(__wt_msg(session,
+			    "%s: log record at position %" PRIu32
+			    " has unused corruption", log_fh->name, offset));
```

Comment:
Please modify to " has unused[%d] corruption %" PRIx8 and print i, logrec->unused[i]